### PR TITLE
Move some image functions into separate file

### PR DESCRIPTION
**Pull request by acoates-ms**
Moving a couple of functions out of ReactImage which is tied to Xaml, so it can be shared with non-xaml code.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/10568)

### CODE REVIEW

File: vnext/Microsoft.ReactNative/Utils/ImageUtils.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+#include <cxxreact/JSBigString.h>
+#include <future>
```
Comment (aeulitz) [inline vnext/Microsoft.ReactNative/Utils/ImageUtils.h:6]: nit: Are those headers needed to understand the decls in this file?

File: vnext/Microsoft.ReactNative/Utils/ImageUtils.h [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  double width = 0;
+  double height = 0;
+  double scale = 1.0;
+  bool packagerAsset = false;
+  ImageSourceType sourceType = ImageSourceType::Uri;
+  ImageSourceFormat sourceFormat = ImageSourceFormat::Bitmap;
+};
+
+winrt::Windows::Foundation::IAsyncOperation<winrt::Windows::Storage::Streams::IRandomAccessStream>
+GetImageMemoryStreamAsync(ReactImageSource source);
```
Comment (aeulitz) [inline vnext/Microsoft.ReactNative/Utils/ImageUtils.h:28]: >ReactImageSource source [](http://example.com/codeflow?start=26&length=23)

Seems like a hefty thing to pass by value.

**Review by aeulitz — APPROVED** [review]
:shipit:

File: vnext/Microsoft.ReactNative/Utils/ImageUtils.h (hunk 1 above)
Comment (acoates-ms) [reply vnext/Microsoft.ReactNative/Utils/ImageUtils.h]: Removed

**Pull request merged**